# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mupdf/hotfix-make.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- mupdf-1.24.10-source/Makerules.vanilla	2024-10-02 13:10:33.000000000 -0400
+++ mupdf-1.24.10-source/Makerules	2024-11-05 22:44:33.550000005 -0500
@@ -275,8 +275,8 @@
       SYS_GLUT_CFLAGS := $(shell pkg-config --cflags glut gl)
       SYS_GLUT_LIBS := $(shell pkg-config --libs glut gl)
     else
-      SYS_GLUT_CFLAGS :=
-      SYS_GLUT_LIBS := -lglut -lGL
+      SYS_GLUT_CFLAGS := $(shell pkg-config --cflags glut gl)
+      SYS_GLUT_LIBS := $(shell pkg-config --libs glut gl)
     endif
   endif
 
